yaml/pois.yaml

```diff
@@ -1,16 +1,424 @@
+  # urban, rural, residential, farmland - no POIs
+
+  ############################################################
+  # TIER 3
+  ############################################################
+  # aerodrome - no POIs
+  # military
+  - filter: {landuse: military}
+    min_zoom: { min: [ { max: [ 8, { lit: zoom + 2 }, { lit: *tier3_min_zoom } ] }, 14 ] }
+    output:
+      kind: military
+      tier: 3
+  # university
+  - filter: {amenity: university}
+    min_zoom: { min: [ { max: [ { lit: zoom + 2.55 }, { lit: *tier3_min_zoom } ] }, 15 ] }
+    output: {kind: university, tier: 3}
+  # college
+  - filter: {amenity: college}
+    min_zoom: { min: [ { max: [ { lit: zoom + 2.35 }, { lit: *tier3_min_zoom } ] }, 16 ] }
+    output: {kind: college, tier: 3}
+
+  ############################################################
+  # TIER 4
+  ############################################################
+  # breakwater - no POIs
+  # bridge - no POIs
+  # cemetery
+  - filter: {landuse: cemetery}
+    min_zoom: { min: [ { max: [ 12, { lit: zoom + 3 }, { lit: *tier4_min_zoom } ] }, 14 ] }
+    output:
+      kind: cemetery
+      tier: 4
+  # commercial - no POI
+  # cutline - no POI
+  # dike - no POI
+  # generator
+  - filter: {power: generator}
+    min_zoom: { max: [15, { lit: *tier4_min_zoom } ] }
+    output: {kind: generator, tier: 4}
+  # golf_course
+  - filter: {leisure: golf_course}
+    min_zoom: { min: [ { max: [ 12, { lit: zoom + 2 }, { lit: *tier4_min_zoom } ] }, 14 ] }
+    output:
+      kind: golf_course
+      tier: 4
+  # groyne - no POI
+  # hospital
+  - filter: {amenity: hospital}
+    min_zoom: { min: [ { max: [ { lit: zoom + 3.32 }, { lit: *tier4_min_zoom } ] }, 14 ] }
+    output: {kind: hospital, tier: 4}
+  # industrial - no POI
+  # pier - no POI
+  # plant
+  - filter: {power: plant}
+    min_zoom: { min: [ { max: [ 12, { lit: zoom + 3 }, { lit: *tier4_min_zoom } ] }, 14 ] }
+    output:
+      kind: plant
+      tier: 4
+  # railway - no POI
+  # recreation_ground
+  - filter: {landuse: recreation_ground}
+    min_zoom: { min: [ { max: [ 12, { lit: zoom + 3 }, { lit: *tier4_min_zoom } ] }, 14 ] }
+    output:
+      kind: recreation_ground
+      tier: 4
+  # retail - no POI
+  # sports_centre
+  - filter: {leisure: sports_centre}
+    min_zoom: { min: [ { max: [ { lit: zoom + 3.98 }, { lit: *tier4_min_zoom } ] }, 17 ] }
+    output:
+      kind:
+        expr: CASE WHEN tags ? 'sport' AND tags->'sport' IN ('fitness', 'gym') THEN 'fitness' ELSE 'sports_centre' END
+      sport:
+        expr: CASE WHEN tags ? 'sport' AND tags->'sport' IN ('fitness', 'gym') THEN NULL ELSE tags->'sport' END
+      tier: 4
+  # stadium
+  - filter: {leisure: stadium}
+    min_zoom: { min: [ { max: [ { lit: zoom + 2.3 }, { lit: *tier4_min_zoom } ] }, 15 ] }
+    output: {kind: stadium, tier: 4}
+  # substation
+  - filter: {power: substation}
+    min_zoom: 15
+    output:
+      kind: substation
+      tier: 4
+  # tower - no POI
```

Comment:
The corresponding feature in `landuse` isn't a `power=tower`, but a `man_made=tower`. Do we want to conflate the two?